### DISCUSSION TITLE
Random connectivity issues after enabling ipv6 for the internal network

### DISCUSSION THREAD

**Discussion by szaimen** [nextcloud/all-in-one#2557] [Wiki]
Apparently Docker has still some issues with ipv6.

I found this which explains some random dns resolution issues that appear from time to time after enabling ipv6:
https://github.com/moby/moby/issues/41651

---

You should be able to fix this by so:
1. Stop your containers from the AIO interface
2. After the docker daemon started correctly run `sudo docker network disconnect nextcloud-aio nextcloud-aio-mastercontainer` to disconnect the mastercontainer from the nextcloud-aio network.
3. now remove the nextcoud-aio network by running `sudo docker network rm nextcloud-aio`
4. Then run `sudo docker network create --driver bridge nextcloud-aio` in order to recreate the nextcloud-aio network
5. Finally run `sudo docker network connect nextcloud-aio nextcloud-aio-mastercontainer` to connect the mastercontainer to the network again.
6. run `sudo docker restart nextcloud-aio-mastercontainer`
7. At last, open the AIO interface and start your containers again

**Comment by iGadget** [comment]
Even though I don't remember enabling ipv6, I am running into random connectivity issues. So I went through al the steps mentioned above... But unfortunately the issue remains - random 'Connection lost' issues when editing .md files in via browser (both through the Files and the Collectives app).

**Reply by szaimen** [reply]
Please open a new discussion for this. Thanks!
